### PR TITLE
DAT-4021 Maintenance script to enable Europa Theme on wikis without infoboxes

### PR DESCRIPTION
**Pull request by sebastianmarzjan**
## DESCRIPTION

This script likely doesn't require a merge. Submitting PR for code review and to make testing easier.

### CODE REVIEW

**Comment by RafLeszczynski** [comment]
LGTM

File: extensions/wikia/PortableInfobox/maintenance/EnableEuropaTheme.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ */
+
+require_once( getenv( 'MW_INSTALL_PATH' ) !== false ? getenv( 'MW_INSTALL_PATH' ) . '/maintenance/Maintenance.php' : dirname( __FILE__ ) . '/../../../../maintenance/Maintenance.php' );
+
+class EnableEuropaTheme extends Maintenance {
+	public function __construct() {
+		parent::__construct();
+
+		$this->mDescription = 'Enable Europa Theme on wikis that do not use Portable Infoboxes yet';
+		$this->addOption( 'make-changes', 'Do real delete. By default performs a dry run.' );
```
Comment (idradm) [inline extensions/wikia/PortableInfobox/maintenance/EnableEuropaTheme.php]: Do I sense copy-pasting here?

Comment (sebastianmarzjan) [inline extensions/wikia/PortableInfobox/maintenance/EnableEuropaTheme.php]: Yes, though this is 'copy from clipboard w/ older version there' ;-)
tackled in 17033cd

**Comment by idradm** [comment]
LGTM

**Pull request closed without merging**